pages/_app.js

```diff
@@ -1,18 +1,38 @@
 import App from 'next/app';
 import GlobalStyle from '@catho/quantum/GlobalStyle';
+import { Hide } from '@catho/quantum';
 
 export default class HomePage extends App {
   componentDidMount() {
     const hiddenMenu = document.getElementById('hiddenMenu');
     document.getElementById('menuButton').addEventListener('click', showMenu);
+    const mobileMenu = document.getElementById('MobileMenu');
```

Comment:
```suggestion
    const mobileMenu = document.getElementById('MobileMenu');
    const menuButton = document.getElementById('menuButton');

    menuButton.addEventListener('click', showMenu);
```
Putting a menuButton in a named variable makes the code a little easier to read